app/assets/javascripts/application.js

```diff
@@ -1,3 +1,4 @@
+//= require jquery
```

Comment:
This is a big new requirement - do we absolutely need it/ what are we using it for?

Reply:
Hi @theseanything - I totally understand.  The issue is the Jasmine tests - from the original find support app - require it!  And as we're simply copying over...

Reply:
So my worry is that this means jquery is now loaded by all smart answers, regardless of whether they use them. Can we tweak the code to not use jquery?

Reply:
I don't believe the escape_link code or tests use JQuery at all.  So I'm a tad confused too.  It seems to be a Jasmine requirement as without it you get `Uncaught ReferenceError: $ is not defined`!

Reply:
Removed the jQuery code and removed the dependency.